dev/com.ibm.ws.jca_fat_regr.cfg.prop/fat/src/suite/r80/base/jca16/TestSetupUtils.java

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package suite.r80.base.jca16;
+
+import org.jboss.shrinkwrap.api.ArchivePath;
+import org.jboss.shrinkwrap.api.Filter;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+
+/**
+ * Class <CODE>TestSetupUtils</CODE> provides utility methods used
+ * to help set up resources needed for tests
+ */
+public class TestSetupUtils {
+
+    public static final String raDir = "test-resourceadapters/adapter.regr/resources/META-INF/";
+
+    private static JavaArchive resourceAdapterEjs_jar;
```

Comment:
When we repeat tests using Jakarta features.  I wonder if caching the resource adapter like this will make it so the transformer doesn't transform the app. 

At first glance I don't think it will.  I believe the transformer comes in when we use `ShrinkHelper.exportToServer(server, path, app, ... options)`.

It might be worthwhile to test this locally by repeating a single test class for EE9 and make sure that the app gets transformed.

Reply:
I was able to repeat the test class that uses this class for EE9 and it transformed and ran successfully so think we should be good.